src/lib.rs

```diff
@@ -7,3 +7,14 @@ pub mod python;
 pub mod report;
 pub mod stacktrace_constants;
 pub mod util;
+
+// This macro merges all [&str] slices into single Vec<String>.
+#[macro_export]
+macro_rules! concatall {
```

Comment:
Maybe we put this macro to stacktrace_constants/utill? And think about better name?

Reply:
done